### ISSUE TITLE
Support specifying tags on CreateItem.

### ISSUE THREAD

**Issue by VictorioBerra** [jscarle/OnePassword.NET#49]
As far as I can see, there is no way to set tags when creating a new item.

**Comment by jscarle** [comment]
According to the official [CLI documentation](https://developer.1password.com/docs/cli/create-item/#with-an-item-json-template), there's no mention that the command line options are compatible with reading a template, so I'll have to do some testing.

It all comes down to needing to adjust [this line](https://github.com/jscarle/OnePassword.NET/blob/8e0def4ffc85eb435038eaa1bbf45a69cd28856c/OnePassword.NET/OnePasswordManager.Items.cs#L79). If you have the time to do some tests with the CLI executable, it would speed up implementation of the changes.

**Comment by VictorioBerra** [comment]
@jscarle  Can  you elaborate on what you mean by reading a template?

`PS C:\Program Files\1Password CLI> .\op.exe items get sample1 --vault XYZ --format json`

```json
{
  "id": "XYZ",
  "title": "Sample1",
  "tags": ["LinqPadSecret"],
  "version": 1,
  "vault": {
    "id": "XYZ",
    "name": "XYZ"
  },
  "category": "LOGIN",
  "last_edited_by": "NCHDKEDUENDZPPSWVVSFB7Y6OQ",
  "created_at": "2023-09-22T20:56:10Z",
  "updated_at": "2023-09-22T20:56:10Z",
  "additional_information": "Sample1",
  "fields": [
    {
      "id": "password",
      "type": "CONCEALED",
      "purpose": "PASSWORD",
      "label": "password",
      "value": "Sample1",
      "reference": "op://XYZ/Sample1/password",
      "password_details": {
        "strength": "TERRIBLE"
      }
    },
    {
      "id": "username",
      "type": "STRING",
      "purpose": "USERNAME",
      "label": "username",
      "value": "Sample1",
      "reference": "op://XYZ/Sample1/username"
    },
    {
      "id": "notesPlain",
      "type": "STRING",
      "purpose": "NOTES",
      "label": "notesPlain",
      "reference": "op://XYZ/Sample1/notesPlain"
    }
  ]
}
```

**Comment by jscarle** [comment]
As mentioned, the current command is:

```bash
op item create --vault {vault.Id} -
```

The last dash means read from stdin.

In the documentation, they use the example by piping from the result of a get template command.

```bash
op item template get Login | op item create --vault Tutorial -
```

What we need to figure out is when using the variation of create that uses stdin (finishing with the final dash), if its possible to use other switches like ---tags. It may be possible, it may not, the documentation doesn't make it clear. The order of options may also influence.

**Comment by VictorioBerra** [comment]
@jscarle Got it, I will run some tests Monday and get back to you here.

**Comment by VictorioBerra** [comment]
@jscarle So it looks like it did accept `--title` after the trailing `-`:

![image](https://github.com/jscarle/OnePassword.NET/assets/2934507/432f756c-625e-43ff-a41d-a69710164524)

**Comment by jscarle** [comment]
Thanks for testing this for me. I'll try to get that implemented this week.

**Comment by jscarle** [comment]
Resolved by latest release v2.3.2.
